python/cudf/cudf/core/dataframe.py

```diff
@@ -496,7 +494,7 @@ def _constructor(self):
 
     @property
     def _constructor_sliced(self):
-        return Series
+        return cudf.Series
```

Comment:
Are all of these changes still necessary now that we resolved the circular dependency issue?

Reply:
Not essential in `dataframe.py`, but I think we can directly reference when needed using `cudf.Series` so we don't run into circular Import issues when we try copying docs elsewhere.

Reply:
I imagine DataFrame should depend on Series but Series shouldn't depend on DataFrame. If we're copying docs we should be able to absolutely refer to DataFrame for example.

Reply:
Sure, Made this Change in `dataframe.py`